core/src/wallet.proto

```diff
@@ -132,6 +132,9 @@ message Key {
 
   // Encrypted version of the seed
   optional EncryptedData encrypted_deterministic_seed = 9;
+
+  // The original account path
+  optional string account_path = 10;
```

Comment:
Hmmm, isn't this redundant? Deterministic keys already have their path stored into the protobuf.